src/main/webapp/mapScript.js

```diff
@@ -89,53 +96,66 @@ function addNewTableItem(name, index) {
   tabel.appendChild(newPlace);
 }
 
-function deletePlace(contentId) {
-  let tableItem = document.getElementById("place" + contentId);
+function getActualIndex(placeIndex) {
```

Comment:
nit: the name is a bit weird. Index is also overloaded, so perhaps use placeId rather than placeIndex and call the function something like findIndex.

Reply:
Done